.env

```
@@ -12,3 +12,20 @@ NEXT_PUBLIC_API_URL=http://localhost:3000
 # FIREBASE_AUTH_EMULATOR_HOST=127.0.0.1:9099
 
 APP_BASE_URL=http://localhost:3000
+
+# Socket.IO Configuration
+SOCKET_PORT=3002
+ALLOWED_ORIGINS=http://localhost:3000,http://localhost:3001,http://localhost:3002,http://localhost:3003
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The ALLOWED_ORIGINS key should go before the SOCKET_PORT key

(UnorderedKey)